app/webpacker/styles/components/inset-text.scss

```diff
@@ -7,6 +7,12 @@ section.inset-text {
   &.yellow {
     background-color: $yellow-light;
     border-left-color: $black;
+
+    @at-root (with: rule) {
+      a {
+        @extend .link--black;
```

Comment:
Just checking and I might have misunderstood here, I though the link colour is blue ?

Reply:
Basically, by default the link colour is blue but I am overwriting it using @at-root rule for ```yellow``` background only where it's supposed to stay black. I could add the similar rule to ```grey``` block but no point to overwrite default.

The URLs to check are: https://review-get-into-teaching-app-3336.london.cloudapps.digital/steps-to-become-a-teacher 
and
https://review-get-into-teaching-app-3336.london.cloudapps.digital/non-uk-teachers/teach-in-england-if-you-trained-overseas